task 3.txt

```diff
@@ -0,0 +1,5 @@
+solution :
+-file for managing the features and users (csv or yml)
+-secret manager for storing the password and other needed info of the users
+-creating ps script that install the features and creates the users with their data on the server
```

Comment:
How would the script do this?

Reply:
The script will use AWS CLI, it will conect to the relevant windows server and then pull the files (features and users) from where they are stored.
next it will install the features one by one and after it the script will retrieve for each user their sensitive data (include password) from the secret manager and create it as new user